_posts/2024-05-14-ASUworkshop.md

```diff
@@ -0,0 +1,173 @@
+style="float: right; height: 20em; " />
+
+<img
+src="/public/images/MolSSI_Logo.png"
+title="MolSSI logo" alt="MolSSI logo"
+style="float: right; height: 5em; " />
+
+MDAnalysis, in conjunction with the [Molecular Sciences Software Institute][MolSSI]
+(MolSSI) and with the support of the [Chan-Zuckerberg Inititative][CZI] and the
+[Center for Biological Physics][CBP], is holding a **2-day** workshop on **June
+24th-25th** 2024 at **Arizona State University** in Tempe, Arizona, USA.
+
+This two-day workshop is intended to transform attendees from users to
+developers and will cover an introduction to MDAnalysis, software best
+practices, and guide participants through the creation of their own [MDAKit][mdakits].
+
+### Workshop outline
+
+MDAnalysis developers and experienced MolSSI instructors will teach
+participants how to build their own MDAnalysis-based Python packages by guiding
+them through the development of their own MDAKits. Specifically, the workshop
+will include 3 modules: (1) an introduction to using and writing custom
+analyses in MDAnalysis; (2) an overview of software development and maintenance
+best practices; and (3) an interactive hackathon session where participants
+write their own MDAKits.
+
+The program will run from the morning of Monday, June 24th until the early
+evening of Tuesday June 25th. 
+
+Parts of the workshop will be held in a hybrid mode for those unable to attend
+in-person. Registration for the hybrid components is required. Recordings of
+hybrid sessions will be made available after the workshop.
+
+Breakfast and lunch will will be provided together with tea/coffee during
+breaks.
+
+
+
+### Registration
+
+Attendance at this workshop will be *free* but limited to 40 in-person
+participants.
+
+A number of **bursaries** will be available to assist with travel and
+accommodation costs for participants from outside the Phoenix area. Bursaries
+(up to *$600 in costs for US domestic travel*) and *up to 3 nights of hotel
+accommodation* (single room within 5 minutes of the workshop venue) will be
+awarded.
+
+<a href="https://LINK.TO.REGISTRATIONFORM" target="_blank" style="background:#FF9200;padding:10px;margin:10px 0px;text-align:center;text-decoration:none;font-size:12pt;color:#000000;display:inline-block;border-radius:3px">Register</a>
+
+#### Application *with bursary request* (non-local attendees)
+
+Register by **May 27, 2024** (midnight anywhere on Earth).
+
+We especially welcome your application if you consider yourself being part of
+an under-represented demographic in the computational molecular sciences.
+
+Bursaries are reserved to enable the attendance of participants from outside
+the greater Phoenix area.
+
+Note that airfare for travel awardees will have to be *booked by June 3 through
+our travel agency* so as an awardee you will have to be able to
+respond to our communications (via email) immediately and act quickly.
+
+Bursary recipients will be notified by May 29. 
```

Comment:
```suggestion
Bursary recipients will be notified by May 30. 
```
The timeline works for me (though as a note, May 27 is a Mon, and May 28 is a Tues). I am only changing the date since we say _by_ May XX. That way, we have May 28 to review/decide, and can send notifications out _on_ May 29.

Reply:
I appreciated the detail to semantics. 

I left it as "on May 29 or earlier" to convey that we are keeping it tight. I'd like people to start looking out for notification starting May 29. 

The reg form also contains May 29.